client/js/socket-events/configuration.js

```diff
@@ -30,6 +31,14 @@ socket.on("configuration", function(data) {
 		pop.play();
 	});
 
+	if (data.upload) {
+		$("#upload").show();
```

Comment:
Just realized that this show/hide needs to be moved before `if (options.initialized)`.

Keep the init here though.

Reply:
done